src/main/java/forestry/api/genetics/ISpeciesRoot.java

```diff
@@ -69,14 +69,35 @@
 	IIndividual getMember(ItemStack stack);
 
 	IIndividual getMember(NBTTagCompound compound);
-	
-	<O extends Object, I extends IIndividual> void registerTranslator(Object translatorKey, IIndividualTranslator<I, O> translator);
-	
+
+	/**
+	 * @param translatorKey The key of the translator, by default it is the item or the block of the {@link ItemStack} or
+	 * 						{@link net.minecraft.block.state.IBlockState} that you want to translate with the translator.
+	 * @param translator A translator that should be used to translate the data.
+	 */
+	void registerTranslator(Object translatorKey, IIndividualTranslator translator);
+
+	/**
+	 * @param translatorKey The key of the translator, by default it is the item or the block of the {@link ItemStack} or
+	 * 						{@link net.minecraft.block.state.IBlockState} that you want to translate with the translator.
+	 */
 	@Nullable
-	<O extends Object, I extends IIndividual> IIndividualTranslator<I, O> getTranslator(Object translatorKey);
+	IIndividualTranslator getTranslator(Object translatorKey);
 
+	/**
+	 * Translates {@link net.minecraft.block.state.IBlockState}s and {@link net.minecraft.item.ItemStack}s into genetic
+	 * data.
+	 */
 	@Nullable
-	<O extends Object, I extends IIndividual> I translateMember(O objectToTranslator);
+	<I extends IIndividual> I translateMember(Object objectToTranslator);
+
+	/**
+	 * Translates {@link net.minecraft.block.state.IBlockState}s and {@link net.minecraft.item.ItemStack}s into genetic
+	 * data.
+	 *
+	 * @since Forestry 5.8
+	 */
+	ItemStack getGeneticEquivalent(Object objectToTranslator);
```

Comment:
I don't really like that these work on `Object`, can you make them work on `ItemStack` and `IBlockState` parameters as separate functions? We will not have any other types so I don't think it needs to be very generic.